app/views/course/assessment/assessments/show.html.slim

```diff
@@ -77,8 +77,12 @@
     = render partial: @assessment.assessment_conditions.includes(:conditional), suffix: 'condition'
 
   - if can?(:attempt, @assessment) && @assessment.folder.materials.present?
-    h3 = t('.files')
-    = render partial: 'layouts/materials', locals: { folder: @assessment.folder }
+    - enable_link = !current_component_host[:course_materials_component].nil?
+    h3 = t('.files') if enable_link || can?(:manage, @assessment)
+    = render partial: 'layouts/materials',
+      locals: { folder: @assessment.folder,
+                enable_link: enable_link,
```

Comment:
Can you test if the following works:

```suggestion
                enable_link,
```

Not sure whether it behaves the same way as js in slim files.

Reply:
Nope, it doesnt work for the slim file.

Reply:
Ok sure, thanks for checking!